server/meeting_test.go

```diff
@@ -67,9 +67,9 @@ func TestPlugin_GenerateHashtag(t *testing.T) {
 				meeting: &Meeting{
 					ChannelID:     "QA Backend",
 					Schedule:      []time.Weekday{time.Monday},
-					HashtagFormat: "QA-{{January 02 2006}}",
+					HashtagFormat: "QA_{{January 02 2006}}",
 				}},
-			want:    "#QA-" + assertNextWeekdayDate(time.Monday, true).Format("January 02 2006"),
+			want:    "#QA_" + strings.ReplaceAll(assertNextWeekdayDate(time.Monday, true).Format("January 02 2006"), " ", "_"),
```

Comment:
Why does `#QA-` need to be changed to `#QA_` here?

Reply:
Reason:
**Our plugin should be database agnostic.**


<img width="880" alt="Screenshot_87" src="https://user-images.githubusercontent.com/85980820/132450192-a142f5b5-be3c-4cf4-b65f-35cb3d826942.png">
Ref. https://docs.mattermost.com/messaging/searching-in-mattermost.html

Reply:
@hanzei It seems we should be avoiding dashes and dots in all hashtags throughout plugins